weareinreach/inreach-catalog
# Feature/1197433510679486/fix map toggle

## Description

Adds functionality to toggle map view on/off to generalized murder button

- Asana ticket: https://app.asana.com/0/1132189118126148/1197433510679486/f

## PR Checklist





## How to Test

1. Navigate to AC catalog home page
2. Select 'Mexico' from country dropdown list
3. Verify that Generalized crime map is displayed by default
4. Click 'Generalized crime' button
5. Verify that Generalized crime map is no longer displayed
6. Click 'Generalized crime' button again
7. Verify that Generalized crime map is displayed
